dev/tasks/r/github.linux.versions.yml

```diff
@@ -30,7 +30,6 @@ jobs:
         r_version:
           # We test devel, release, and oldrel in regular CI.
           # This is for older versions
-          - "3.6"
           - "4.0"
           - "4.1"
```

Comment:
Should we add 4.2 as replacement for 3.6?

Reply:
Ah, yes, yes good catch!